### ISSUE TITLE
log of training

### ISSUE THREAD

**Issue by HuangJunJie2017** [mks0601/PoseFix_RELEASE#27]
Sincerely thanks for your contribution!
Would it is presumptuous of me to ask you to provide the log file of training process?
As my training with this repository didn't converge.
I used the following common and change nothing of the respository:
python train.py --gpu 0-7
here is part of the log
[train_logs.txt](https://github.com/mks0601/PoseFix_RELEASE/files/3629788/train_logs.txt)

**Comment by mks0601** [comment]
This is very hard to see. Please summarize your problem.

**Comment by HuangJunJie2017** [comment]
During the training process, the train loss did't converge and the effectivity of the algorithm became worser and worser

**Comment by HuangJunJie2017** [comment]
some time the loss become very large:
![image](https://user-images.githubusercontent.com/31001081/65235967-2e08df00-db0a-11e9-99e2-85a83a6d080b.png)
and some time being negative:
![image](https://user-images.githubusercontent.com/31001081/65236081-66a8b880-db0a-11e9-9176-8351793a3e09.png)

**Comment by mks0601** [comment]
Hmm. I haven't met that problem. Which dataset are you using?

**Comment by HuangJunJie2017** [comment]
COCO

**Comment by mks0601** [comment]
Your downloaded code is the most recent version?

**Comment by mks0601** [comment]
Did you change

```
    ## dataset
    dataset = 'PoseTrack' # 'COCO', 'PoseTrack', 'MPII'
    testset = 'val' # train, test, val (there is no validation set for MPII)
```

to your desired settings? (COCO, train)

**Comment by HuangJunJie2017** [comment]
I downloaded the code on the 8th of August, I suppose it is the latest code

**Comment by HuangJunJie2017** [comment]
dataset = 'COCO' # 'COCO', 'PoseTrack', 'MPII'
    testset = 'val' # train, test, val (there is no validation set for MPII)
I did't change the testset and suppose it is a params only for testing process

**Comment by mks0601** [comment]
Right. It seems the number of iteration for one epoch is smaller than mine. Do you use other version of COCO? Anyway, I haven't met this problem, so I cannot give you clear answer. Sorry. I seems there is some issue with heatmap loss. Could you debug yourself and let me know any progress? Thank you!

One thing I want to check is your trained model gives bad performance because of this issue?

**Comment by HuangJunJie2017** [comment]
ok, as i have modified the code as discussion in issue #8 to fixed the bug of InvalidArgumentError
Could you please provide me with the change of loss during the training process?

**Comment by mks0601** [comment]
Sorry I don't keep training log :(

**Comment by HuangJunJie2017** [comment]
Never mind, thanks for your patience!
